src/gsy_e/models/area/scm_manager.py

```diff
@@ -187,7 +185,7 @@ def calculate_home_energy_bills(self, home_uuid: str) -> None:
 
         # Next case is the consumption. In this case the energy allocated from the community
         # is sufficient to cover the energy needs of the home.
-        if home_data.allocated_community_energy_kWh > home_data.energy_need_kWh:
+        if home_data.allocated_community_energy_kWh >= home_data.energy_need_kWh:
```

Comment:
@spyrostz I wanted to discuss this with you (already started a discussion on slack). What do you think about this change?

Reply:
Not a problem and looks good to me! It should not have a huge effect in the operation of the SCM in general since for this to happen, the community consumption has to exactly match the production of a single home, therefore it is very infrequent.